www/templates/html/scripts/frontend.js

```diff
@@ -0,0 +1,13 @@
+require(['jquery'], function($){
+	WDN.initializePlugin('modal', [function() {
```

Comment:
I contributed real feedback!!! 

I think this is how you can do it:
`require(['jquery','modal'], function($,modal){
modal.initialize();
// colorbox stuff
});`